src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AbstractEnchantmentMachine.java

```diff
@@ -31,13 +35,15 @@
 
     private final ItemSetting<Boolean> useLevelLimit = new ItemSetting<>(this, "use-enchant-level-limit", false);
     private final IntRangeSetting levelLimit = new IntRangeSetting(this, "enchant-level-limit", 0, 10, Short.MAX_VALUE);
+    private final ItemSetting<List<String>> ignoredLores = new ItemSetting<>(this, "ignored-lores", Arrays.asList("&7- &cCan't be used in " + this.getItemName()));
```

Comment:
I feel like this should be an empty list not gonna lie... I don't see a reason to do any checks when likely this will not be changed by the vast majority of servers.

Reply:
If we do want to keep a value here though, this should really be using Collections.singletonList

Reply:
Collections.singletonList can cause error in ItemSetting process before.. Let's keep it empty simply.

Reply:
Or maybe I will add a toggle setting, as BusyBiscuit suggested.